WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -138,3 +143,17 @@ inline static BOOL isSnapshotTypeAmongstGivenTypes(XCElementSnapshot* snapshot,
   }
   return NO;
 }
+
+inline static BOOL doesSnapshotHasMoreThanOneVisibleChildrenSnapshots(XCElementSnapshot* snapshot)
+{
+  NSArray<XCElementSnapshot *> *cellSnapshots = [snapshot fb_descendantsMatchingType:XCUIElementTypeCell];
```

Comment:
I know. And thats why following lines where it will just take all available descendants. Actually I copied this from here https://github.com/facebook/WebDriverAgent/blob/master/WebDriverAgentLib/Categories/XCUIElement%2BFBScrolling.m#L92-L97